modules/blocks.php

```diff
@@ -274,24 +277,28 @@ function jetpack_business_hours_render( $attributes, $content ) {
 			if ( $today === $day ) {
 				$now = strtotime( current_time( 'H:i' ) );
 				if ( $now < $opening ) {
-					$content .= '<br />';
-					$content .= esc_html( sprintf(
-						/* Translators: Amount of time until business opens. */
+					$days_hours .= '<br />';
+					$days_hours .= esc_html( sprintf(
+					/* Translators: Amount of time until business opens. */
 						_x( 'Opening in %s', 'Amount of time until business opens', 'jetpack' ),
 						human_time_diff( $now, $opening )
 					) );
 				} elseif ( $now >= $opening && $now < $closing ) {
-					$content .= '<br />';
-					$content .= esc_html( sprintf(
-						/* Translators: Amount of time until business closes. */
+					$days_hours .= '<br />';
+					$days_hours .= esc_html( sprintf(
+					/* Translators: Amount of time until business closes. */
 						_x( 'Closing in %s', 'Amount of time until business closes', 'jetpack' ),
 						human_time_diff( $now, $closing )
 					) );
 				}
 			}
-		} else {
-			$content .= esc_html__( 'CLOSED', 'jetpack' );
+			$days_hours .= '<br />';
+		}
+
+		if ( empty( $days_hours ) ) {
+			$days_hours = esc_html__( 'CLOSED', 'jetpack' );
```

Comment:
Kinda nitpick but to me, it feels like this should be just "Closed" and capitalized in CSS. Gives more possibilities for folks to change styling via CSS if they want.